doc/cprover-manual/quantifiers.md

```diff
@@ -0,0 +1,40 @@
+# Quantifiers
+
+### Definition
+
+`__CPROVER_forall{}` and `__CPROVER_exists{}` are both supported in function-contracts context.
+
+```
+__CPROVER_forall { *type identifier* : range; *expression* }
+__CPROVER_exists { *type identifier* : range; *expression* }
```

Comment:
I am not sure how to interpret the syntax here. How about:

```c
__CPROVER_forall { *identifier* *type*; *boolean expression* }
__CPROVER_exists { *identifier* *type*; *boolean expression* }
```

Reply:
Way better. Replace it.